src/tests/suite/daos_obj.c

```diff
@@ -3165,9 +3165,9 @@ tgt_idx_change_retry(void **state)
 
 		/** exclude target of the replica */
 		print_message("rank 0 excluding target rank %u ...\n", rank);
```

Comment:
minor, may be worth changing the print_message to say "excluding rank" rather than "excluding target rank"

Reply:
Done.